core/src/main/java/com/netflix/conductor/metrics/WorkflowMonitor.java

```diff
@@ -74,51 +74,60 @@ public WorkflowMonitor(
     }
 
     @Scheduled(
-            initialDelayString = "${conductor.workflow-monitor.stats.initial-delay:120000}",
-            fixedDelayString = "${conductor.workflow-monitor.stats.delay:60000}")
+            initialDelayString = "${conductor.workflow-monitor.stats.initial-delay:-1}",
+            fixedDelayString = "${conductor.workflow-monitor.stats.delay:-1}")
     public void reportMetrics() {
-        try {
-            if (refreshCounter <= 0) {
-                workflowDefs = metadataService.getWorkflowDefs();
+        if (refreshCounter <= 0) {
+                // workflowDefs = metadataService.getWorkflowDefs();
                 taskDefs = new ArrayList<>(metadataService.getTaskDefs());
                 refreshCounter = metadataRefreshInterval;
-            }
-
-            getPendingWorkflowToOwnerAppMap(workflowDefs)
-                    .forEach(
-                            (workflowName, ownerApp) -> {
-                                long count =
-                                        executionDAOFacade.getPendingWorkflowCount(workflowName);
-                                Monitors.recordRunningWorkflows(count, workflowName, ownerApp);
-                            });
-
-            taskDefs.forEach(
-                    taskDef -> {
+        }
+
+        // Commented out as we don't have use case for Pending Workflows as of now and hence we don't have pending workflow implementation from
```

Comment:
- Remove these code if they are no longer needed
- We can always get them from the version control